pkg/utils/names/names.go

```diff
@@ -18,8 +18,21 @@ package names
 
 import (
 	"fmt"
+	"strings"
+)
+
+const (
+	FloatingAddressIPClaimNameSuffix = "floating-ip-address"
 )
 
 func GetDescription(clusterName string) string {
 	return fmt.Sprintf("Created by cluster-api-provider-openstack cluster %s", clusterName)
 }
+
+func GetFloatingAddressIPAddressClaimName(openStackMachineName string) string {
```

Comment:
Could we simplify this? The double "address" is making my head spin :sweat_smile: 
```suggestion
func GetFloatingIPAddressClaimName(openStackMachineName string) string {
```

Reply:
The names aren't great, but that's where I ended up..

TL;DR
Floating IP is not used by openstack so we ended up with `FloatingAddressFromPool` instead of `FloatingIPFromPool` and the claim kind is `IPAddressClaim`, so the most descriptive "correct" naming is the `...FloatingAddressIPAddressClaim..` but I'd be happy to make it less verbose.

I think I'd prefer `..FloatingAddressClaim..` over `..FloatingIPAddressClaim..` would that work for you given the initial reasoning?

Reply:
Right, so this is to make it super clear that the name is for an `IPAddressClaim` that is filled by a `FloatingAddress`?
I think `GetFloatingAddressClaimName` would make sense, like you said. That also matches with `GetOpenStackMachineNameFromClaimName` below.